src/DonorProfiles/Repositories/Donations.php

```diff
@@ -196,7 +196,7 @@ protected function getPaymentInfo( $payment ) {
 		return [
 			'amount'   => $payment->subtotal,
 			'currency' => $payment->currency,
-			'fee'      => $payment->fees,
+			'fee'      => ( $payment->total - $payment->subtotal ),
```

Comment:
For my edification, why are you favouring a calculation over an existing property? I don't know the benefit/requirement here.

Reply:
I'm performing a calculation here because, despite being documented, a `fees` property is not actually accessible on the Give_Payment class. A quick search for `->fees` or even `fees` reveals that this logic is not centralized in the codebase as I was originally led to believe.